extension/data_loader/CMakeLists.txt

```diff
@@ -24,6 +24,11 @@ if(NOT ET_HAVE_SYS_MMAN_H AND NOT WIN32)
        "extension/data_loader/mmap_data_loader.cpp"
   )
 endif()
+if(WIN32)
```

Comment:
Why is this necessary? I thought scott already selectively included this source file in build whatever.bzl

Reply:
Yeah, I chatted with Scott briefly on this, and his recommendation was to manage it locally. The main build variables list has to be mechanically parsable from both CMake and Buck, so it can't include more complex conditional logic.